selfdrive/test/process_replay/process_replay.py

```diff
@@ -353,3 +380,33 @@ def replay_process(cfg, lr):
 
         recv_cnt -= m.which() in recv_socks
   return log_msgs
+
+def cpp_replay_process(config, logreader):
+  pub_sockets = [s for s in config.pub_sub.keys()]  # We dump data from logs here
+  sub_sockets = [s for _, sub in config.pub_sub.items() for s in sub]  # We get responses here
+  pm = messaging.PubMaster(pub_sockets)
+  sockets = {s : messaging.sub_sock(s, timeout=1000) for s in sub_sockets}
+
+  all_msgs = sorted(logreader, key=lambda msg: msg.logMonoTime)
+  pub_msgs = [msg for msg in all_msgs if msg.which() in list(config.pub_sub.keys())]
+
+  manager.start_managed_process(config.proc_name)
```

Comment:
currently manager builds on import, but let's not rely on that behavior
```suggestion
  manager.prepare_managed_process(config.proc_name)
  manager.start_managed_process(config.proc_name)
```